.mailmap

```diff
@@ -1313,3 +1313,5 @@ zzj <29055749+zjzh@users.noreply.github.com>
 Øyvind Jensen <jensen.oyvind@gmail.com>
 Łukasz Pankowski <lukpank@o2.pl>
 彭于斌 <1931127624@qq.com>
+Yashvardhan Prasad<yash19b2@gmail.com>yasphy<yash19b2@gmail.com>
+Yashvardhan Prasad<yash19b2@gmail.com>Yashvardhan Prasad <78342519+yasphy@users.noreply.github.com>
```

Comment:
You should include spaces like this:
```
Yashvardhan Prasad <yash19b2@gmail.com> yasphy<yash19b2@gmail.com>
Yashvardhan Prasad <yash19b2@gmail.com> Yashvardhan Prasad <78342519+yasphy@users.noreply.github.com>
```
After adding these if you run `bin/mailmap_check.py` then it will reorder the file because the .mailmap file is kept in alphabetical order.

Reply:
What about Release Notes? Why am I facing issue?

Reply:
And the .mailmap error isn't resolving. How do I run the commands in the web version?